src/triage/experiments/base.py

```diff
@@ -628,6 +629,10 @@ def feature_dicts(self):
         values being lists of feature names
 
         """
+        if not self.master_feature_dictionary:
+            logger.warning("No features have been created. Either there is no feature configuration" \
+                           "or there was some problem processing them.")
+            return []
```

Comment:
Is this meant to be part of this PR -- it seems a little unrelated? Or just a separate check that we should add?

Reply:
This had to do with the test_partial_experiments::TestLabels failure that came up early in this PR.

FAILED src/tests/test_partial_experiments.py::Labels::test_run - ValueError: ("Problem! The feature group definition {'all': [True]} did not find any matches", 'in feature dictionary {}')

I've forgotten why this error didn't show up before the other changes in this PR, but I think this check makes sense in general; if "'only send labels' and call run()" is a valid partial experiment use case, we likely want this code to just throw a warning like the one I added here.

Reply:
Thanks @thcrock -- that makes sense (though agree it's also a bit surprising the error wasn't coming up before)